.travis.yml

```diff
@@ -1,28 +1,25 @@
+sudo: false
+dist: trusty
+
 language: php
 
 php:
-    - 5.5
     - 5.6
     - 7.0
+    - 7.1
     - hhvm
 
-env:
-    - SYMFONY_VERSION=2.3.*
-    - SYMFONY_VERSION=2.4.*
-    - SYMFONY_VERSION=2.5.*
-    - SYMFONY_VERSION=2.6.*
-    - SYMFONY_VERSION=2.7.*
-    - SYMFONY_VERSION=2.8.*
-    - SYMFONY_VERSION=3.0.*
```

Comment:
I don't think it's necessary to test each symfony version because of their BC policy. So testing with the composer flag `--prefer-lowest` will give us symfony 2.7.* components and with a normal update it will give us the highest stable component versions. So this way we got everything covered with minimal tests. If I don't overlook anything of course :)